resolver-dns/src/main/java/io/netty/resolver/dns/DnsQueryContext.java

```diff
@@ -78,14 +85,26 @@ DnsQuestion question() {
         return question;
     }
 
+    List<DnsRecord> additional() {
```

Comment:
@Karry last nit but can we just remove this method ? I think we not need it at al and exposing it if not needed makes no sense. After this is addressed I can pull in. (Please ensure you also squash and sign the ICLA: http://netty.io/s/icla).